### PR TITLE
 document sidebar layout for workflow manager

### PR DESCRIPTION
**Pull request by FrankXuuu**

### CODE REVIEW

File: src/tactic/ui/tools/pipeline_wdg.py [hunk 1]
```diff
@@ -6613,3 +6618,409 @@ def execute(self):
...
+class DocumentInlineInputWdg(BaseRefreshWdg):
+
+    def get_display(self):
+
+        top = self.top
+
+        return top
+
+
+class PipelineSetCateogryCmd(Command):
```
Comment (CeltonMcGrath) [inline src/tactic/ui/tools/pipeline_wdg.py]: typo!

File: src/tactic/ui/tools/pipeline_wdg.py [hunk 2]
```diff
@@ -269,7 +272,8 @@ def get_display(self):
 
             expression = self.kwargs.get("expression")
 
-            pipeline_list = PipelineListWdg(save_event=save_event, save_new_event=save_new_event, settings=self.settings, expression=expression )
+            #pipeline_list = PipelineListWdg(save_event=save_event, save_new_event=save_new_event, settings=self.settings, expression=expression )
+            pipeline_list = PipelineDocumentWdg()
```
Comment (CeltonMcGrath) [inline src/tactic/ui/tools/pipeline_wdg.py]: add a project setting

File: src/tactic/ui/tools/pipeline_wdg.py [hunk 3]
```diff
@@ -6613,3 +6618,394 @@ def execute(self):
...
+                {
+                    "type": "group",
+                    "state": "on",
+                    "group_level": 1,
+                    "title": "Job",
+                },
+                {   
+                    "type": "sobject",
+                    "group_level": 2,
+                    "expression": "@SEARCH(sthpw/pipeline['category','Job']['@ORDER_BY','name']['project_code', '%s'])" % (project_code)
```
Comment (CeltonMcGrath) [inline src/tactic/ui/tools/pipeline_wdg.py]: why is category job?

Comment (CeltonMcGrath) [reply src/tactic/ui/tools/pipeline_wdg.py]: I think that the original PipelineListWdg has seperated pipelines by search type, which is why you are seeing Job. I think you can remove this.

File: src/tactic/ui/tools/pipeline_wdg.py [hunk 4]
```diff
@@ -6613,3 +6618,394 @@ def execute(self):
...
+                    "group_level": 1,
+                    "title": "Templates",
+                },
+                {
+                    "type": "sobject",
+                    "group_level": 2,
+                    "expression": "@SEARCH(sthpw/pipeline['category','Template']['@ORDER_BY','name']['project_code', '%s'])" % (project_code)
+                },
+            ]
+        }
```
Comment (CeltonMcGrath) [inline src/tactic/ui/tools/pipeline_wdg.py:6639]: We should have another category for uncategorized workflows.

File: src/tactic/ui/tools/pipeline_wdg.py (hunk 3 above)
Comment (FrankXuuu) [reply src/tactic/ui/tools/pipeline_wdg.py]: I just took the default category names for these. 

Forgot to mention that if we commit these changes, we have to input default categories for the pipelines we already have.

File: src/tactic/ui/tools/pipeline_wdg.py [hunk 5]
```diff
@@ -6613,3 +6618,394 @@ def execute(self):
...
+            height="auto",
+            width="100%",
+            view="workflow",
+            #drag_action_script="spme/workflow_document_drag_action",
+            extra_data={
+                "min_height": 14,
+                #"font_size": 10,
+                "single_line": "true",
+                #"mode": pipeline_type,
+                #"group_level_padding": 15,
```
Comment (CeltonMcGrath) [inline src/tactic/ui/tools/pipeline_wdg.py]: remove these comments

**Pull request merged**